rules/cross-platform/lateral_movement_remote_file_creation_in_sensitive_directory.toml

```diff
@@ -27,16 +27,18 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-file where (event.action == "creation" or event.action == "modification") and
-process.name:("System", "scp", "sshd", "smbd", "vsftpd", "sftp-server") and not
-user.name:("SYSTEM", "root") and
-(file.path : ("C*\\Users\\*\\AppData\\Roaming*", "C*\\Program*Files\\*",
-    "C*\\Windows\\*", "C*\\Windows\\System\\*",
-    "C*\\Windows\\System32\\*", "/etc/*", "/tmp*",
-    "/var/tmp*", "/home/*/.*", "/home/.*", "/usr/bin/*",
-    "/sbin/*", "/bin/*", "/usr/lib/*", "/usr/sbin/*",
-    "/usr/share/*", "/usr/local/*", "/var/lib/dpkg/*",
-    "/lib/systemd/*"
+file where event.action : ("creation", "modification") and
+  not user.name:("SYSTEM", "root") and
+  process.name:("System", "scp", "sshd", "smbd", "vsftpd", "sftp-server") and
```

Comment:
I'd always try to use the `in` statement rather than the `:`, but not sure whether that's possible in these two, as `in` is case sensitive. Not familiar enough with the Windows logs.

Reply:
We can do it as the system process & user should keep that case, but normal executables have variations, not sure if this could make performance differences tho